game/managers/event_manager.py

```diff
@@ -0,0 +1,12 @@
+_handlers = {}
+class EventManager:
+
+	def add_handler(event, handler):
+		if handler not in _handlers:
+			_handlers.setdefault(event, [])
+			_handlers[event] = [handler]
```

Comment:
This code doesn't allow to add more than one handler.
This if makes sure that we have a default value for this event - an empty list.
Then you need to append the handler outside the if